src/SharpCompress/Archives/IArchiveExtractionListener.cs

```diff
@@ -1,8 +1,6 @@
-using SharpCompress.Common;
-
 namespace SharpCompress.Archives;
 
-internal interface IArchiveExtractionListener : IExtractionListener
+internal interface IArchiveExtractionListener
```

Comment:
This interface isn't needed any more.  The same mechanism should be used for Archives